src/open_inwoner/pdc/tests/test_product_location.py

```diff
@@ -227,3 +227,48 @@ def test_exception_is_handled_when_city_and_postcode_are_not_provided(
             ],
         )
         mock_geocoding.assert_called_once_with(" ,  ")
+
+
+class TestLocationDetailView(WebTest):
```

Comment:
I would collapse these three test functions into one, like they are so simple this just generates more code.

Then you can group statements with some comments like `# adding one published and one draft product`, and later `# only the published product shows` and `# check location details`